### PR TITLE
Task/admin my settings page profile tab

### PR DESCRIPTION
**Pull request by egiegarciano14**
## Related Links

- Task Link:
- https://app.asana.com/0/1202252431396579/1202448426751470/f
- https://app.asana.com/0/1202252431396579/1202434821847348/f
- Route Link: http://localhost:3000/admin/my-settings

## Description
- Create breadcrumb
- Create admin my settings page profile tab

## Notes

N/A

## Screenshots
image

### CODE REVIEW

File: src/pages/admin/MySettings/index.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      <div className='mb-5 flex text-brown-900'>
+        <Breadcrumb text='Dashboard' to='/admin/dashboard' />
+        <Breadcrumb text='My Settings' last />
+      </div>
+      <section>
+        <div className='text-2xl font-bold text-brown-900'>My Settings</div>
+        <hr className='mt-3 border border-orange-900' />
+        <div className='mx-8 mt-8 flex'>
+          <div className='flex flex-col items-center'>
+            <img
```
Comment (CaneteJoash) [inline src/pages/admin/MySettings/index.js]:
```suggestion
            <LazyLoadImage
```
Can we use LazyLoadImage?

File: src/pages/admin/MySettings/index.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+              src='/images/Test05.webp'
+              className='mb-10 h-[294px] w-[294px] rounded-full'
+              alt='profile'
+            />
+            <div>
+              <input type='file' name='image' ref={fileInput} style={{ display: 'none' }} />
+              <button
```
Comment (CaneteJoash) [inline src/pages/admin/MySettings/index.js]:
```suggestion
              <Button
                IsCustomizable= {true}
                className='rounded-md bg-black px-5 py-2 font-semibold text-white'
                onClick={() => fileInput.current.click()}
                buttonLabel = 'Upload Photo'
              />
```
something like the suggestion in the top. 
Please use the shared button component and pass a style prop.
we can create switch case in  the Button component to identify the default button and the customize one.
please pass also the onclick attribute in the button component as a prop

File: src/pages/admin/MySettings/Profile.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+            type='number'
+            value='number'
+            placeholder='000000'
+            onChange={handleOnChange}
+          />
+        </div>
+        <hr className='my-10 border border-orange-900' />
+        <label htmlFor='aboutme' className='flex flex-col text-xl font-bold'>
+          ABOUT ME
+          <textarea
```
Comment (CaneteJoash) [inline src/pages/admin/MySettings/Profile.js]: is it possible to add this into the InputField?

**Review by ErickDoroy — COMMENTED** [review]
- Please update the margin of those in the red box
Screenshot 2022-06-17 135708
Screenshot 2022-06-17 135853

**Review by ErickDoroy — APPROVED** [review]

**Pull request merged**